src/equipment/misc/markers.md

```diff
@@ -2,4 +2,27 @@
 
 ## Marker Refill Instructions
 
```

Comment:
Having it be in this format would be easier to read:

1. step
2. step
    - comment